charts/joplin-server/values.yaml

```diff
@@ -0,0 +1,28 @@
+# Default values for pyLoad.
+
+image:
+  repository: joplin/server
+  pullPolicy: IfNotPresent
+  tag: 1.7.2
+
+strategy:
+  type: Recreate
+
+service:
+  port:
+    port: 22300
+
+env:
+  # APP_BASE_URL: https://joplin.domain
```

Comment:
Could you annotate this values? 
https://github.com/norwoodj/helm-docs

Reply:
Ok, I added annotation like you did on [librespeed chart](https://github.com/angelnu/charts/blob/master/charts/librespeed/values.yaml)